typescript-conditionals/main.ts

```diff
@@ -0,0 +1,139 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+function isUnderFive(num: number): boolean {
+  return num < 5;
+}
+
+function isEven(num: number): boolean {
+  if (num % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function startsWithJ(str: string): boolean {
+  const firstChar = str[0];
+  switch (firstChar) {
+    case 'J':
+      return true;
+    default:
+      return false;
+  }
+}
+
+interface Person {
+  name: string;
+  age: number;
+}
+
+function isOldEnoughToDrink(person: Person): boolean {
+  return person.age >= 21;
+}
+
+function isOldEnoughToDrive(person: Person): boolean {
+  // The below line is to satisfy our compiler's desire that every parameter has a use
+  person.name += ' the Great';
+  return false;
+}
+
```

Comment:
You are missing a function here. You have `isOldEnoughToDrink` and `isOldEnoughToDrive` but are missing `isOldEnoughToDrinkAndDrive` function. Even if it should always return `false`, you should still have a condition statement for practice!